jumpscale/sals/fs/__init__.py

```diff
@@ -389,20 +389,36 @@ def is_empty_dir(path: str) -> bool:
 is_binary_file = lambda path: not is_ascii_file(path)
 
 
-def is_broken_link(path: str, clean=False) -> bool:
+def is_broken_link(path: str) -> bool:
     """Checks if path is a broken symlink
 
     Args:
         path (str): path to check
-        clean (bool, optional): remove symlink if broken. Defaults to False.
 
-    Raises:
-        NotImplementedError: [description]
+    Returns:
+        bool:   True if path is a broken symlink
+                False if path not found or symlink is not broken
+    """
+    if os.path.islink(path) and not exists(path):
```

Comment:
we can return boolean expression directly

```suggestion
    return os.path.islink(path) and not exists(path)
```